libpagekite/pkcommon.h

```diff
@@ -143,6 +143,7 @@ typedef unsigned int              uint32_t;
 #    define SSL_OP_NO_COMPRESSION 0
 #  endif
 #  define PKS_DEFAULT_CIPHERS "HIGH:!aNULL:!eNULL:!LOW:!MD5:!EXP:!PSK:!SRP:!DSS"
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
I was having problems with this in another project, is this enough here to cover also the "version x, with compat turned off" ?  there's also OPENSSL_COMPAT_xxx defines that might work better?

Reply:
Should be fine. These functions are noop anyways. Using TLS_method also allows to simplify the code a bit.